lib/std/zig/system/linux.zig

```diff
@@ -344,6 +391,9 @@ pub fn detectNativeCpuAndFeatures() ?Target.Cpu {
         .powerpc, .powerpcle, .powerpc64, .powerpc64le => {
             return PowerpcCpuinfoParser.parse(current_arch, f.reader()) catch null;
         },
+        .riscv64, .riscv32 => {
```

Comment:
Is `riscv32` like `aarch64_32`?

Reply:
I'm not sure. I haven't heard of any 32-bit RISC-V systems which can run Linux as well.

Reply:
I'm asking to learn if we should collapse `riscv32` and `riscv64` into one prong but perhaps that is OK.

Reply:
Oh, I'm not sure heh.